cmd/state-custom-installer/util.go

```diff
@@ -0,0 +1,152 @@
+package main
```

Comment:
Nit: a generic util file like this is a breeding ground for unorganized code. Recommend splitting the logic contained into separate files with distinctive naming.